docs/commands/doctor.md

```
@@ -0,0 +1,288 @@
+Ensure required tools are in your PATH:
+```bash
+export PATH=$PATH:/usr/local/bin:$HOME/.local/bin
+```
+
+### Engram unreachable
+Start Engram or configure as MCP server:
+```bash
+engram serve  # or configure in your MCP client
+```
+
+### Low disk space
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)